tensorflow/python/ops/array_ops.py

```diff
@@ -470,7 +470,7 @@ def _SliceHelper(tensor, slice_spec, var=None):
     else:
       begin.append(s)
       end.append(s + 1)
-      strides.append(1)
+      strides.append(constant_op.constant(1, s.dtype))
```

Comment:
hm.  is s assumed to be a numpy element here?  or could it be a regular integer?

should this be `np.ones_like(s)` instead?